notebooks/01.00-How-to-Get-Started.ipynb

```diff
@@ -138,21 +138,36 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Step 7: Open up “01.01-PyRosetta-Google-Drive-Setup.ipynb” in Google Colab by right-clicking, going under “Open With…” and clicking Google Colaboratory.\n",
+    "**Step 7:** Open up “01.01-PyRosetta-Google-Drive-Setup.ipynb” in Google Colab by right-clicking, going under “Open With…” and clicking Google Colaboratory.\n",
     "\n",
-    "Step 8: Follow steps to install PyRosetta. Note: You only need to do this once since this installation will be associated with your Google Drive account.\n",
+    "**Step 8:** Follow steps to install PyRosetta. Note: You only need to do this once since this installation will be associated with your Google Drive account.\n",
     "\n",
-    "Step 9: In “01.01-PyRosetta-Google-Drive-Setup.ipynb”, follow the steps to install third-party external packages, using `pip`:\n",
+    "**Step 9:** In “01.01-PyRosetta-Google-Drive-Setup.ipynb”, follow the steps to install third-party external packages, using `pip`:\n",
     "\n",
     "`pip install biopython blosc dask dask-jobqueue distributed graphviz jupyter matplotlib numpy pandas py3Dmol scipy seaborn traitlets --user`\n",
     "\n",
-    "or using `conda`:\n",
+    "**Step 9.1:**\n",
     "\n",
-    "`conda env create -f environment.yml`\n",
+    "**For Chapter 15, Running PyRosetta in Parallel, you will need to use a specific version of PyRosetta that is built for parallelization.**  This is the serialization build.  Besides manually building it from the Rosetta C++ source code, the general way to obtain this is through the use of a `conda` environment. \n",
```

Comment:
Not every notebook that uses the distributed stuff needs serialization at the moment.  2.08 doesn't, a lot of the others don't as well.  I will see which ones need it and which ones don't.

Reply:
A lot of the notebooks that originally needed that layer, I modified so that they don't anymore - so that the layer would pretty much only be in one chapter.   I'll go through each one to make sure though.

Reply:
I added it to the ones that needed it and switched the Kernel.  Ideally, one would setup the conda env, and then activate before running all the notebooks, just like we did at Code School.  Without this, we are asking for trouble...